x-pack/plugins/cloud_security_posture/public/test/mock_server/handlers/api/mocks/cloud_security_posture.json

```diff
@@ -0,0 +1,14242 @@
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "dfc17731-aa8f-5ecc-878b-113d1db009ca",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "dfc4b9b5-43dc-5ec2-97b4-76a71621fa40",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "e06f9ef1-eedb-5f95-b8d4-36d27d602afd",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "e073f962-74d9-585b-ae5a-e37c461e9b7c",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "e1b73c05-5137-5b65-9513-6f8018b6deff",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "e1c469c1-89d2-5cbd-a1f1-fe8f636b151f",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "e2306922-4f95-5660-bf2e-9610f556de69",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "e24bf247-bfdc-5bbf-9813-165b905b52e6",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "e3c6b85b-703e-5891-a01f-640d59ec449e",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "e570dc22-4f5d-51db-a193-983cb7d20afe",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "e833e6a8-673d-56b2-a979-f9aa4e52cb71",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "e83a8e8a-e34b-5a01-8142-82d5aef60cab",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "e92ddce9-3cba-5e3d-adac-53df0350eca0",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "ea3378aa-250e-50d8-9260-ff8237cf09a2",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "eb9e71ae-113b-5631-9e5c-b7fdc0b0666e",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "ec7949d4-9e55-5f44-8c4a-a0e674a2a46f",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "ed797ade-c473-5b6a-b1e2-1fd4410f7156",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "eda32e5d-3684-5205-b3a4-bbddacddc60f",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "edccbc31-3c4d-5d38-af6a-7fd1d9860bff",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "ede1488a-e8cd-5d5f-a25d-96c136695594",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "ee5ea7e5-dcb4-5abc-ab8c-58b3223669ce",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "eeb00e89-7125-58e8-9248-b9f429583277",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "eed3e284-5030-56db-b749-01d7120dc577",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "ef3852ff-b0f9-51d5-af6d-b1b1fed72005",
+          "type": "csp-rule-template"
+        },
+        {
+          "id": "efec59bf-4563-5da7
```

Comment:
let's name it by the name of the handler it is used for, or at least make it clear that this is about `fleet`. without looking into it I couldn't tell what is it and opening 14k on Github is painful :) if we move to `fleet/` dir as I mentioned in another comment, wouldn't be a big problem, but still I'd like more clear name